docs/source/physics-wiki/physics-index.rst

```diff
@@ -0,0 +1,41 @@
+Physics Getting Started
+=======================
+
+The physics subordinate engine (also known as **Diviner**) of Legion-Engine is a powerful way to handle physics events,
+collisions and other interesting physics related problems for your game.
+
+To use Diviner make sure to link against ``legion-physics.lib``
+If you are on Windows you can do this by right-clicking your project -> Properties -> Linker -> Input 
+`More Details <https://stackoverflow.com/a/4446085/5487005>`_
+
+Adding Physics Components
+-------------------------
+
+
```

Comment:
Need some input from @Developer-The-Great here

Reply:
looks pretty good, I'll add the adding physics component part later